launch_testing/launch_testing/junitxml.py

```diff
@@ -83,7 +83,8 @@ def unittestCaseToXml(test_result, test_case):
     class needs to be a launch_testing.TestResult class
     """
     case_xml = ET.Element('testcase')
-    case_xml.set('classname', type(test_case).__name__)
+    full_classname = type(test_case).__module__ + '.' + type(test_case).__name__
```

Comment:
@hidmic Sorry, I know this question is late to the party, but this will always be launch_testing, right?

I used the module name of the test description function https://github.com/ApexAI/apex_rostest/blob/c8cd0ac6dc5383334fdbfd67e7bfeee62bf37c1f/apex_launchtest/apex_launchtest/loader.py#L97

Reply:
Follow-up I see in a later commit that you did basically the same thing by generating the test name from the name of the test file.  That's what I get for reviewing one commit at a time.

Reply:
> Sorry, I know this question is late to the party, but this will always be launch_testing, right?

It'll be the name of the package the test is in, which may be provided by a caller or be the base name of the test file.